codecarbon/emissions_tracker.py

```diff
@@ -75,6 +75,13 @@ def __init__(
         elif cpu.is_rapl_available():
             logger.info("CODECARBON : Tracking Intel CPU via RAPL interface")
             self._hardware.append(CPU.from_utils(self._output_dir, "intel_rapl"))
+        
+        # Print warning if no supported hardware is found'
+        if not self._hardware:
+            logger.warning('CODECARBON : No CPU/GPU tracking mode found. This '\
+                'may be due to your code running on Windows WSL, or due to '\
+                'unsupported hardware (see '\
+                'https://github.com/mlco2/codecarbon#infrastructure-support)')
```

Comment:
```suggestion
            logger.warning(
                "CODECARBON : No CPU/GPU tracking mode found. This "
                "may be due to your code running on Windows WSL, or due to "
                "unsupported hardware (see "
                "https://github.com/mlco2/codecarbon#infrastructure-support)"
            )
```

Reply:
Looks like the linter was tripping this up. Use this and we can merge!

Reply:
@LiamConnell done as requested, see latest commit, thank you!